request.go

```diff
@@ -729,3 +884,108 @@ func (req *UpsertRequest) BodyFunc(res SchemaResolver) (func(enc *msgpack.Encode
 		return fillUpsert(enc, spaceNo, req.tuple, req.ops)
 	}, nil
 }
+
+// CallRequest helps you to create a call request object for execution
+// by a Connection.
+type CallRequest struct {
+	baseRequest
+	function string
+	args     interface{}
+}
+
+// NewCallRequest returns a new empty CallRequest. It uses request code for
+// Tarantool < 1.7.
+func NewCallRequest(function string) *CallRequest {
+	req := new(CallRequest)
+	req.requestCode = CallRequestCode
+	req.function = function
+	req.args = []interface{}{}
+	return req
+}
+
+// Args sets the args for the call request.
+// Note: default value is empty.
+func (req *CallRequest) Args(args interface{}) *CallRequest {
+	req.args = args
+	return req
+}
```

Comment:
Shouldn't we provide something convenient for adding positional and named binding parameters?

CC @vr009.

Reply:
(It is about `ExecuteRequest`, I placed the comment to the wrong place.)

Reply:
AFAIU you mean something like this:
```go 
ExecuteReqObj.Args([]interface{uuid.New(), "Value_A", 14})

...
...

ExecuteReqObj.PosArg(1, uuid.New()).PosArg(2, "Value_B").NamedArg("count", 15)
```
That looks to be useful when it is needed to change some field explicitly.

Reply:
> That looks to be useful when it is needed to change some field explicitly.

For Execute, we can make special Request Objects, as examples:

https://gobuffalo.io/documentation/database/querying/
https://v1.gorm.io/docs/query.html
https://v1.gorm.io/docs/query.html#Scan
https://v1.gorm.io/docs/query.html#Group-amp-Having

The only thing is that it can be difficult to implement.

Reply:
I'm ok with the current implementation, so I'm not insisting.

Reply:
I guess that nothing will prevent us to add more construction methods for binding parameter in a future. So I'll resolve the discussion. This PR is not where we plan to dig into SQL a lot.